tests/test_simulation.py

```diff
@@ -764,6 +756,35 @@ def test_eff_noise(matrices):
         )
 
 
+@pytest.mark.parametrize(
+    "noise_sample,",
+    [
+        ("depolarizing"),
+        ("dephasing"),
+    ],
+)
+def test_three_level_noise(noise_sample, matrices):
+    np.random.seed(123)
+    reg = Register3D.from_coordinates([(0, 0, 0), (0, 0, 1)], prefix="q")
```

Comment:
Why `Register3D`? The position coordinates of the atoms and the dimensions are actually not related